### PR TITLE
FM2-613: Remove unnecessarily packaged modules

### PR DESCRIPTION
**Pull request by ibacher**
## Description of what I changed

This removes `liquibase-api` and `commons-io` from the OMOD, both of which are supplied by the platform and should resolve the issue with the FHIR2 activator.

Thanks @mogoodrich for pointing out the bug!

## Issue I worked on

see https://issues.openmrs.org/browse/FM2-613

## Checklist: I completed these to help reviewers :)

- [ ] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [ ] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [ ] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [ ] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [ ] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

**Review by mogoodrich — APPROVED** [review]
Approving this one, but there's another follow-on error I'm seeing that I've noted on the ticket.  Thanks @ibacher 

Will wait on merging this one?

File: pom.xml [hunk 1]
```diff
@@ -238,10 +238,18 @@
                         <groupId>org.springframework</groupId>
                         <artifactId>spring-messaging</artifactId>
                     </exclusion>
+                    <exclusion>
+                        <groupId>org.springframework.batch</groupId>
+                        <artifactId>spring-batch-core</artifactId>
+                    </exclusion>
```
Comment (mseaton) [inline pom.xml:244]: @ibacher - I had to add in the above exclusion to get past the next error that @mogoodrich reported.  With this in place, I was able to successfully build/test, and to install the module into a 2.6.3 environment without any errors at startup.  Let me know what you think and hopefully this is ok and we can merge.

**Review by mherman22 — APPROVED** [review]
LGTM, tested it on platform 2.6.3 and everything looks okay and looking through the lib folder, the spring jars (the root cause) have been excluded.

Thanks @mseaton

File: pom.xml (hunk 1 above)
Comment (ibacher) [reply pom.xml:244]: Thanks @mseaton for that catch! This dependency management part is my least favorite part of the module.

**Pull request merged**